canonical-web-and-design/maas.io
# Add /docs/search with the search module

Fixes https://github.com/canonical-web-and-design/base-squad/issues/619

## QA

Get hold of the "Custom Search API key" from [the Google cloud API credentials page](https://console.cloud.google.com/apis/credentials?project=ubuntu-search-1530889417216) - if you don't have access to that page, ask @nottrobin or someone for access.

Then, run the site with:

``` bash
./run --env SEARCH_API_KEY={the-api-key}
```

Now go to http://0.0.0.0:8006/docs, search for something and see the search results at e.g. http://0.0.0.0:8006/docs/search?q=maas.

(Note: The styling will look whack - this is acceptable for now, this is just a proof of concept)
